src/lib/components/tx/modal/index.tsx

```diff
@@ -49,7 +51,7 @@ export const TxModal = ({ result, onClose }: TxModalProps) => {
           {result.receiptInfo.headerIcon}
           {result.receiptInfo.header}
         </ModalHeader>
-        {isUpdateAdminSucceed && <ModalCloseButton color="gray.600" />}
+        {showCloseButton && <ModalCloseButton color="gray.600" />}
```

Comment:
I recall that some tx success modal shouldn't have close button?

Reply:
decided to change